project.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<p>No data preprocessing? Justification for not normalising?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-42FraMeWork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>